pepper-apis/src/main/java/org/broadinstitute/ddp/pex/TreeWalkInterpreter.java

```diff
@@ -199,6 +203,19 @@ private boolean applyFormPredicate(InterpreterContext ictx, FormPredicateContext
         }
     }
 
+    private boolean evalStudyQuery(InterpreterContext ictx, StudyQueryContext ctx) {
+        String umbrellaStudyGuid = extractString(ctx.study().STR());
+        return applyStudyPredicate(ictx, ctx.studyPredicate(), umbrellaStudyGuid);
+    }
+
+    private boolean applyStudyPredicate(InterpreterContext ictx, StudyPredicateContext predCtx, String umbrellaStudyGuid) {
+        Optional<GovernancePolicy> policy = ictx.getHandle().attach(StudyGovernanceDao.class).findPolicyByStudyGuid(umbrellaStudyGuid);
+        String msg = "Governance policy for " + umbrellaStudyGuid + " doesn't exist";
+        return policy.map(
+                p -> p.hasReachedAgeOfMajority(ictx.getHandle(), new TreeWalkInterpreter(), ictx.getUserGuid())
+        ).orElseThrow(() -> new PexFetchException(new NoSuchElementException(msg)));
```

Comment:
Should we consider a missing governance policy an exception or should we default it the `hasAgedUp()` to `false` instead?

Reply:
My 2cents is to default it to `false`. What do other folks think?

Reply:
I’d lean towards throwing an exception- `false` in that case feels like it’s trying to communicate too much information. This isn’t a case where there’s just missing data- the person building the study is attempting to use functionality which they may not have configured, which seems like an exceptional state to me.

Reply:
I also would encourage an exception in this case.  This might move some work onto the client to first verify that they're operating in the context of a study that has age-up rules, but I think it's worth it.  I'm doing something similar when checking aged-up-ness in the invitation registration code to make sure that we see an error when a client passes in an `invitationId` with a registration for a study that has no study governance.

Reply:
Ok, thank you guys, so let's throw an exception.